pkg/plugins/resources/temurin/spec.go

```diff
@@ -0,0 +1,20 @@
+package temurin
+
+type Spec struct {
+	// ReleaseLine specifies the type of Temurin Release to retrieve: 'lts' (default), feature.
```

Comment:
What would you think to split the documentation on mutliple line
```suggestion
	// ReleaseLine specifies the type of Temurin Release to retrieve.
	//
	// default: "lts"
	// 
	// Allowed values:
	// * "lts"
	// * "feature"
```

Reply:
Can you help me / remind me how is this multiline syntax works?

I'll update the other spec attributes on the same model in any case